lib/index.js

```diff
@@ -291,3 +292,28 @@ function formatOutput(data) {
 
   return formatedData;
 }
+
+/**
+ * ensure command exists
+ *
+ * @param cmd {array} parse command + arguments array, e.g,
+ *  [ '/Users/foo/dev/git/ps/test/with',
+ *    'space/sleep',
+ *     'with',
+ *     'space',
+ *     '1'
+ *  ];
+ * @returns {string} the command.  cmd is modified to only contain actual arguments.
+ */
+function ensureCommand(cmd) {
+  var command = cmd.shift();
+  var fs = require('fs');
+
```

Comment:
Still, I would not change the `cmd` directly, I would either get a copy of cmd or just use an `index` to do the shifting.